pickhardtpayments/OracleLightningNetwork.py

```diff
@@ -1,5 +1,8 @@
-from .ChannelGraph import ChannelGraph
-from .OracleChannel import OracleChannel
+from typing import List
+
+from pickhardtpayments.Channel import Channel
```

Comment:
I am pretty sure the old one `from .ChannelGraph import Channel` is what we want. But maybe I have trouble understanding imports in python.